spec/features/user_signup_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.feature "Sign up", type: :feature do
+  scenario "User can sign up" do
+    visit "/welcome"
+    click_button "Sign Up"
+    fill_in "Username", with: "JooMin"
```

Comment:
JooMin spelt differently in this file compared to the others so could possibly cause confusion or issues later down the line when creating more tests?

Reply:
Changed username field to keep it consistent